### PR TITLE
fix(withFeature): disable-failed-tests-screenshots-by-default

### PR DESCRIPTION
**Pull request by zetmate**
Added an env variable to enable/disable failed tests screenshots, also made disabled by default.
Related [this](https://github.com/orgs/wixplosives/projects/27/views/19?pane=issue&itemId=46322716)

### CODE REVIEW

**Comment by zetmate** [comment]
> Why we want to disable this by default?
@barak007 I've heard from @plisovyi that locally it's very annoying that they get generated constantly every time you run the tests that fail.

**Comment by zetmate** [comment]
@barak007 actually it seems like I forgot to put it into gitignore... at least it's not in the codux nor engine

File: packages/test-kit/src/with-feature.ts [hunk 1]
```diff
@@ -225,7 +227,7 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}): WithF
         devtools = envDebugMode ? debugMode : undefined,
         slowMo,
         persist,
-        takeScreenshotsOfFailed = true,
+        takeScreenshotsOfFailed = process.env.TESTS_SCREENSHOTS === 'true',
```
Comment (Ari-Kishon) [inline packages/test-kit/src/with-feature.ts:230]: Maybe "TEST_SCREENSHOTS"

Also lets add some documentation for this environment variable

**Comment by plisovyi** [comment]
If it's in .gitignore, I guess it will be fine, but it makes it required on the consumer's repo side for such a default 🤷🏼

**Comment by barak007** [comment]
we already have few directories that should be ignored that the engine generates. we could make it more verbose but with proper docs it is not an issue.
I don't mind the env variable I just question the default. 
it's my opinion if someone has other strong opinions we can talk it out.

**Comment by plisovyi** [comment]
I'm personally okay with just updating codux .gitignore, and updating docs in the engine about it.

**Comment by barak007** [comment]
Please open new one if you still want the env variable. I think we don't change the default flag

**Pull request closed without merging**